source/chapters/advanced_topics.rst

```diff
@@ -106,6 +106,2785 @@ send messages to text displays on the controller.
+
+   :range: -3.0..3.0
+   :feedback: Waveform
+
+
+The Sampler controls
+~~~~~~~~~~~~~~~~~~~~
+
+Sample decks ("samplers") in Mixxx are identical to decks, they simply
+have a different purpose (playing samples). Any control listed above for
+``[ChannelN]`` will work for a sampler, just replace ``[ChannelN]`` with
+``[SamplerN]``.
+
+.. mixxx:control:: [SamplerN],[Samplers],show_samplers
+
+
+.. mixxx:control:: [SamplerN],[Sampler],SaveSamplerBank
+
+   :range: binary
+   :feedback: Shows Sampler bank(s)
+
+   Save sampler configuration. Make currently loaded tracks in samplers instantly available at a later point.
+
+   :range: binary
+   :feedback: Opens file dialog. Configuration file can be named and saved.
+
+
+.. mixxx:control:: [SamplerN],[Sampler],LoadSamplerBank
+
+   Load saved sampler configuration file and add tracks to the available samplers.
+
+   :range: binary
+   :feedback: Opens file dialog. Select configuration file.
+
+
+Auxiliary controls
+~~~~~~~~~~~~~~~~~~
+
+You can map audio interface's inputs to mixxx's auxiliary input channels and
+connect external audio source to it (cellphone, mp3 player). Then you can use
+your midi controller to control its volume and some other parameters
+(orientation, gain, volume), apply effects and use listen function.
+
+
+Preview Deck controls
+~~~~~~~~~~~~~~~~~~~~~
+
+Preview decks in Mixxx are identical to regular decks. Any control listed above
+for ``[ChannelN]`` will work for a preview deck, just replace ``[ChannelN]``
+with ``[PreviewDeckN]``.
+
+
+The ``[VinylControl]`` group
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. mixxx:control:: [VinylControl],Toggle
+
+   Moves control by a vinyl control signal from one deck to another if using the single deck vinyl control (VC) feature.
+
+   :range: binary
+   :feedback: If VC isn't enabled on any decks, enable it on the first one we're receiving samples for. If VC is enabled on a single (exclusive) deck, and another deck is setup to receive samples, disable it on the former deck and enable it on the next eligible deck (ordered by deck number). If VC is enabled on multiple decks, don't do anything.
+
+   .. versionadded:: v1.10.0
+
+
+The ``[MicrophoneN]`` group
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below, N=2 up to the number of active microphones. e.g ``[Microphone2]``. For Microphone 1, just use ``[Microphone]``.
+
+.. mixxx:control:: [MicrophoneN],enabled
+
+   1 if a microphone input is enabled, 0 if not.
+
+   :range: binary
+   :feedback: Microphone is enabled.
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],orientation
+
+   Set microphone orientation,
+
+   :range:
+      ===== ===================================
+      Value Meaning
+      ===== ===================================
+      0     Left side of crossfader
+      1     Center (not affected by crossfader)
+      2     Right side of crossfader
+      ===== ===================================
+   :feedback: N/A
+
+   .. versionadded:: v1.10.0
+
+
```

Comment:
The entire section should be removed.
Controls are being created indeed but not processed in the mixer, that's why we don't have orientation controls in the GUI. I just filed https://bugs.launchpad.net/mixxx/+bug/1899639
```suggestion
```

Reply:
I added a deprecation note, so that people who find it in the developer tools don't get confused.